CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+CHANGES IN VERSION 1.21.0 (FROM 1.20.2)
+===================================
+
+Appium 1.21.0 is a minor release
+
+### General
+
+* Minimum NodeJS version moved to 12
+
+### Android General
```

Comment:
Please mention 
- the recent adb fixes for `mobile: type` command:  https://github.com/appium/appium-adb/pull/567
- addition of speed parameter to mock location calls: https://github.com/appium/appium-adb/pull/563

Reply:
note: added `mobile:type`. The speed is not yet.